io-sim/tags

```diff
@@ -0,0 +1,1930 @@
+!_TAG_FILE_SORTED	1	//
```

Comment:
I don't think you want to commit the `tags` file.

Reply:
You are right!